ecs-cli/modules/commands/compose/compose_command.go

```diff
@@ -103,10 +103,7 @@ func createCommand(factory composeFactory.ProjectFactory) cli.Command {
 		Name:   "create",
 		Usage:  "Creates an ECS task definition from your compose file. Note that we do not recommend using plain text environment variables for sensitive information, such as credential data.",
 		Action: compose.WithProject(factory, compose.ProjectCreate, false),
-		Flags: []cli.Flag{
-			command.OptionalClusterFlag(),
-			command.OptionalRegionFlag(),
-		},
+		Flags:  append(command.OptionalClusterAndProfileFlags(), command.OptionalRegionFlag()),
```

Comment:
Oh, seeing this diff, perhaps stating each set of flags like in the old lines 106-109 might actually be better than the suggestion I made above?